app/services/data/get-first-name-by-reference.js

```diff
@@ -0,0 +1,7 @@
+const config = require('../../../knexfile').asyncworker
+const knex = require('knex')(config)
+
+module.exports = function (schema, reference) {
+  return knex(`${schema}.Visitor`).first().where({'Reference': reference})
+    .select('FirstName')
+}
```

Comment:
Retrieve last name using claim ID.  Claimants can have multiple rows in the Visitor table if they have updated their eligibility details when creating a repeat claim.  Claim Id should be a field on the task object